iot/config/connect.rst

```diff
@@ -20,35 +20,42 @@ Follow the steps to connect your IoT Box.
 Ethernet Connection
 ~~~~~~~~~~~~~~~~~~~
 
-1. Connect to the IoT Box all the devices that have to be connected with
-   cables (ethernet, usb devices, etc.).
+* Connect to the IoT Box all the devices that have to be connected with
+  cables (ethernet, usb devices, etc.).
 
-2. Power on the IoT Box.
+* Power on the IoT Box.
 
-3. Then click on the Scan button.
+* Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
 
 .. image:: media/connect04.png
```

Comment:
I would decrease the image size so that it doesn't take the entire page width for 2 words :p